packages/react-ui-kit/src/Form/Select.tsx

```diff
@@ -1,6 +1,6 @@
 /*
  * Wire
- * Copyright (C) 2018 Wire Swiss GmbH
+ * Copyright (C) 2022 Wire Swiss GmbH
```

Comment:
I got told that we dont chage the copyright date, so it should be 2018 I guess

Reply:
is this still 2022?